src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, job_id, initial_params):
         self.energy_trade_profile = {}
         self.energy_trade_profile_redis = {}
         self.file_export_endpoints = FileExportEndpoints()
+        self.load_count = load_count
```

Comment:
Same as below, the load count could be part of the MarketUnmatchedLoads class, and could also potentially even be calculated in its constructor.

Reply:
But I like to add to consideration, that if `ConstSettings.GeneralSettings.KEEP_PAST_MARKETS = True` this function will mot be applied. Or shall I also add it to the ExportUnmatchedLoads class?

Reply:
Hmm, I see the issue now. I would prefer it to add it to the ExportUnmatchedLoads class too if possible. That way the unmatched loads calculations will be isolated in their own module.